src/0_utils.js

```diff
@@ -76,6 +76,17 @@ utils.whiteListSessionData = function(data) {
 	};
 };
 
+utils.cleanLinkData = function(linkData, config) {
+	if (config.WEB_BUILD) {
+		linkData['source'] = 'web-sdk';
+		if (linkData['data']['$desktop_url'] !== undefined) {
+			linkData['data']['$desktop_url'] = linkData['data']['$desktop_url'].replace(/#r:[a-z0-9-_]+$/i, '');
```

Comment:
sorry -- i know this is how it was before -- but gotta clean the branch_match_id out of here too